ddl/ddl.go

```diff
@@ -447,6 +447,9 @@ func checkJobMaxInterval(job *model.Job) time.Duration {
 	if job.Type == model.ActionAddIndex {
 		return 3 * time.Second
 	}
+	if job.Type == model.ActionCreateTable {
```

Comment:
Please add a `TODO` here.

Reply:
Why not consider CreateDatabase/DropTable....?

Reply:
@shenli 
"DropTable" has 4 states. So this operation takes longer than it takes.

Reply:
I am confused by the context. Anyone help me out? 

> request add a TODO.
TODO for what? 
> Why not consider CreateDatabase/DropTable....? 
you mean we reduce tick of `CreateDatabase/DropTable` to 500ms?

Reply:
Yes, they are the same story with CreateTable.
> you mean we reduce tick of CreateDatabase/DropTable to 500ms?

Reply:
@shenli 
This is the estimated time.
```
tidb> create database db;
Query OK, 0 rows affected (0.16 sec)
tidb> create table db.x(a int, b int);
Query OK, 0 rows affected (0.19 sec)
tidb> drop table db.x;
Query OK, 0 rows affected (0.39 sec)
tidb> drop database db;
Query OK, 0 rows affected (0.35 sec)
```